charts/medusa-operator/templates/deployment.yaml

```diff
@@ -11,6 +11,10 @@ spec:
   template:
     metadata:
       labels: {{- include "k8ssandra-common.labels" . | indent 8 }}
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
```

Comment:
This isn't the variable scope you want. It needs to be more narrow.

Reply:
I am not really sure about this comment. Could you explain more about this?